docs/Homebrew-on-Linux.md

```diff
@@ -45,6 +45,9 @@ brew install hello
 
 If you're using an older distribution of Linux, installing your first package will also install a recent version of *glibc* and *gcc*. Use `brew doctor` to troubleshoot common issues.
 
+> [!NOTICE]
```

Comment:
This is GitHub-specific syntax that our website generation doesn't currently support